pcsx2/gui/MainFrame.cpp

```diff
@@ -576,7 +576,7 @@ MainEmuFrame::MainEmuFrame(wxWindow* parent, const wxString& title)
 	// background logo size, which is what it'll eventually be resized to.
 	wxSize backsize(m_background->GetBitmap().GetWidth(), m_background->GetBitmap().GetHeight());
 
-	wxString wintitle = "PCSX2 - 128MB - [27598fe]";
+	wxString wintitle = "PCSX2 - Jak and Daxter Debug Mode";
```

Comment:
Since the .exe is not technically J&D specific, its a good idea to get rid of the commit SHA, but lets just make it something like `PCSX2 - 128MB Build` or something similar.  The patches are what makes it work out of the box for J&D

Reply:
Updated!